frontend/pages/project-developers/new.tsx

```diff
@@ -412,15 +412,15 @@ const ProjectDeveloper: PageComponent<ProjectDeveloperProps, NakedPageLayoutProp
                   register={register}
                   aria-required
                   placeholder={formatMessage({
-                    defaultMessage: 'insert your answer (max 500 characters)',
-                    id: 'rBoq14',
+                    defaultMessage: 'insert your answer (max 600 characters)',
```

Comment:
In the future, I would suggest you keep these changes separated. It's not part of the task to update the other fields.